### PR TITLE
Feature/use flashbots

### PR DESCRIPTION
**Pull request by F4ever**

### CODE REVIEW

File: scripts/utils/variables.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import logging
 import os
+import random
+import string
```
Comment (avsetsin) [inline scripts/utils/variables.py:4]: Are these imports in use?

File: scripts/depositor_utils/depositor_bot.py [hunk 2]
```diff
@@ -265,12 +270,16 @@ def do_deposit(self):
...
             DEPOSIT_FAILURE.inc()
+
         else:
             logger.info({'msg': f'Deposited successfully.', 'value': str(result.logs)})
             SUCCESS_DEPOSIT.inc()
 
-        logger.info({'msg': f'Deposit method end. Sleep for 1 minute.'})
-        time.sleep(60)
+        web3.disconnect()
+        web3.provider = HTTPProvider(variables.WEB3_INFURA_PROJECT_ID)
```
Comment (avsetsin) [inline scripts/depositor_utils/depositor_bot.py:279]: Does `WEB3_INFURA_PROJECT_ID` contain the full url? It looks like only infra token is passed to `HTTPProvider`

File: scripts/utils/variables.py (hunk 1 above)
Comment (F4ever) [reply scripts/utils/variables.py]: Fixed

File: scripts/depositor_utils/depositor_bot.py [hunk 3]
```diff
@@ -250,6 +251,10 @@ def do_deposit(self):
             return
 
         logger.info({'msg': 'Creating tx in blockchain.'})
+
+        web3.disconnect()
+        web3.provider = HTTPProvider('https://rpc.flashbots.net')
```
[the remark this answers is not in the document]
Comment (F4ever) [reply scripts/depositor_utils/depositor_bot.py]: Fixed

**Review by avsetsin — APPROVED** [review]
lgtm

**Pull request merged**